package.json

```diff
@@ -7,7 +7,6 @@
     "start": "npm run build && node dist/app.js",
     "build": "babel server/src -d dist",
     "start:dev": "nodemon server/src/app.js --exec babel-node",
-    "pretest": "export NODE_ENV=test || SET \"NODE_ENV=test\" && npm run migrate",
```

Comment:
Good job @sojida. Is there any reason you removed it? I think the pretest script, although not set up properly right now, has a role to play down the line. most especially when we start having scripts for client side and server side.

Reply:
Keen observation @vic3king. I am aware of its importance but at the moment it is not in use. However, when it is needed it would be included then updated along the way by whosoever requires it.